location/models.py

```diff
@@ -16,6 +18,15 @@
 
 logger = logging.getLogger(__file__)
 
+def free_cache_for_user(user_id):
+    cache_name = f'location_allowed_{user_id}'
+    cache.delete(cache_name)
+    cache_name = f"user_districts_{user_id}"
+    cache.delete(cache_name)
+    
+@receiver(post_save, sender=core_models.InteractiveUser)
+def free_cache_post_user_save(sender, instance, created, **kwargs):
+    free_cache_for_user(instance.id)
```

Comment:
Is it worth checking if `instance.officer.location` changed before invalidating the cache?

Reply:
I don't want to slow the system, loading the is_allowed won't take so much time, the issue is how often it is called also user are not often modified